src/resources/views/fields/browse_multiple.blade.php

```diff
@@ -0,0 +1,174 @@
+@php
+$multiple = array_get($field, 'multiple', true);
+$value = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : null));
+
+if (!$multiple && is_array($value)) {
+    $value = array_first($value);
+}
+@endphp
+
+<div @include('crud::inc.field_wrapper_attributes') >
+
+    <div><label>{!! $field['label'] !!}</label></div>
```

Comment:
Should the field label be non-escaped? (I know, I know - there's no consistency here).

Reply:
This also is copied from the browse field, I do not mind changing it to non escaped. In any case a review of all labels might be in order to make them consistent across all fields.

Reply:
Yeah, fair enough.